pkgs/applications/misc/slade/default.nix

```diff
@@ -1,18 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, wxGTK, gtk2, sfml, fluidsynth, curl, freeimage, ftgl, glew, zip }:
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, wxGTK, gtk2, sfml, fluidsynth
+, curl, freeimage, ftgl, glew, zip, lua, mpg123 }:
 
 stdenv.mkDerivation rec {
   pname = "slade";
-  version = "3.1.1.5";
+  version = "3.2.1";
 
   src = fetchFromGitHub {
     owner = "sirjuddington";
     repo = "SLADE";
     rev = version;
-    sha256 = "0mdn59jm6ab4cdh99bgvadif3wdlqmk5mq635gg7krq35njgw6f6";
+    sha256 = "sha256-KFRX3sfI//Op/h/EfEuAZOY22RO5qNXmvhSksC0aS4U=";
   };
 
   nativeBuildInputs = [ cmake pkg-config zip ];
-  buildInputs = [ wxGTK gtk2 sfml fluidsynth curl freeimage ftgl glew ];
+  buildInputs =
+    [ wxGTK gtk2 sfml fluidsynth curl freeimage ftgl glew lua mpg123 ];
+
+  cmakeFlags = [
+    "-DwxWidgets_CONFIG_EXECUTABLE=${wxGTK}/bin/wx-config"
```

Comment:
This is not cross friendly. Can we use it from PATH instead?

Reply:
do you have any pointers on how to do that?

Reply:
I would need to go into the source code or we use buildPackages.

Reply:
In fact, we only have to add `which` in `nativeBuildInputs`, see [here](https://github.com/sirjuddington/SLADE/blob/master/src/CMakeLists.txt#L20).